.github/workflows/continuous-integration-linux.yml

```diff
@@ -1,16 +1,10 @@
 name: github-Linux
 
 on:
-  push:
-    branches:
-      - develop
-  pull_request:
-    paths-ignore:
-    - '**/*.md'
-    types: [ opened, reopened, synchronize ]
+  workflow_call:
 
 concurrency:
-  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}
+  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}-cilnx
```

Comment:
I am not a big fan of these made up suffixes
`-cilnx`
`-cilnx32`
`-cilnxhpx`
`-ciosx`
`-ciwnds`

The "CI" part does not seem super useful
If you are going to abbreviate Windows, you should probably say "WIN" instead of "WNDS"